### DISCUSSION TITLE
How to control remotely?

### DISCUSSION THREAD

**Discussion by hacker-DOM** [pear-devs/pear-desktop#1723] [Q&A] [answered]
I would like to use Hammerspoon to play/pause Youtube Music. I think osascript or URI are the best methods, I read that `youtubemusic://COMMAND` was implemented. However, I am not able to get it working from a browser (COMMAND = PAUSE). The app opens, but the music is not paused

**Comment by inson1** [comment]
@hacker-DOM 
> I read that youtubemusic://COMMAND was implemented. However, I am not able to get it working from a browser (COMMAND = PAUSE. The app opens, but the music is not paused

I didnt know that was implemented (at least the opening). May you send me link, where did you read it? :) 

There should be REST api in future. https://github.com/th-ch/youtube-music/issues/1674

After most of the issues are done, mobile app for controling the app would be dope.

**Reply by ArjixWasTaken** [reply, marked as answer]
[These](https://github.com/th-ch/youtube-music/blob/master/src/providers/song-controls.ts#L7-L37) are the available commands, more specifically:

`previous`, `next`, `playPause`, `like`, `dislike`, `go10sBack`, `go10sForward`, `go1sBack`, `go1sForward`, `shuffle`, `switchRepeat`, `volumeMinus10`, `volumePlus10`, `fullscreen`, `muteUnmute`, `play`, `pause`

And they are case-sensitive, I assume @hacker-DOM used uppercase letters, so it did not work.
One example command is `youtubemusic://pause`
